src/subdomains/core/history/mappers/transaction-dto.mapper.ts

```diff
@@ -183,6 +187,33 @@ export class TransactionDtoMapper {
       date: tx.created,
     };
   }
+
+  private static mapFees(entity: BuyCryptoExtended | BuyFiatExtended): FeeDto {
+    const referencePrice = entity.inputAmount / entity.inputReferenceAmount;
+
+    return {
+      rate: entity.percentFee,
+      fixed:
+        entity.absoluteFeeAmount != null &&
+        Util.roundReadable(entity.absoluteFeeAmount * referencePrice, isFiat(entity.inputAssetEntity)),
+      min:
+        entity.minFeeAmount != null &&
+        Util.roundReadable(entity.minFeeAmount * referencePrice, isFiat(entity.inputAssetEntity)),
+      network:
+        entity.blockchainFee != null &&
+        Util.roundReadable(entity.blockchainFee * referencePrice, isFiat(entity.inputAssetEntity)),
+      dfx:
+        entity.totalFeeAmount != null &&
+        entity.blockchainFee != null &&
+        Util.roundReadable(
+          (entity.totalFeeAmount - entity.blockchainFee) * referencePrice,
+          isFiat(entity.inputAssetEntity),
+        ),
+      total:
+        entity.totalFeeAmount != null &&
+        Util.roundReadable(entity.totalFeeAmount * referencePrice, isFiat(entity.inputAssetEntity)),
+    };
```

Comment:
Haha, this is not working as expected 😉 
![image](https://github.com/DFXswiss/api/assets/85513542/2db40561-1d93-4d69-a484-852996673ebf)

Reply:
Hehe yeah already saw that 😅